### DISCUSSION TITLE
How can I protect ALL mobs (hostile included) and not just passive ones?

### DISCUSSION THREAD

**Discussion by Noxillio** [GriefPrevention/GriefPrevention#1829] [Q&A]
Players should not be able to kill hostile mobs from other players' grinders.

**Comment by RoboMWM** [comment]
there's a legacy FAQ about this. I guess I should try to fold those into the current FAQ too. See wiki page.

**Reply by Noxillio** [reply]
Thank you for your reply.

We need a solution that would help us keep players from killing hostile mobs on other players' claims. The FAQ you are referring to does not mention this, though it does mention hostile mobs and WorldGuard. It is difficult finding a solution for this issue, but it is necessary. Lots of players have grinders for hostile mobs, and it isn't okay that other players are able to come in and ruin the stacks of mobs that have been generated by AFK'ing players for themselves.

**Reply by Jikoo** [reply]
Have they considered building an enclosure around the kill chamber? If you go to all the trouble of building a grinder, it's not hard to build an additional little box with a door.
